lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -138,7 +150,18 @@ serveWallet (cfg, tr) databaseDir hostPref listen lj beforeMainLoop = do
             let seqTl = newTransactionLayer @n (getGenesisBlockHash bp)
             rndApi <- apiLayer tr rndTl (toWLBlock <$> nl)
             seqApi <- apiLayer tr seqTl (toWLBlock <$> nl)
-            startServer tr nPort bp rndApi seqApi
+
+            -- StakePool
+            PoolSqlite.withDBLayer cfg tr (poolDBPath databaseDir) $
+                \splDB -> do
+
+                    -- TODO: I believe @KTorZ wants us to use @WorkerRegistry@
```

Comment:
Not `WorkerRegistry`. There's no need for this. But `newWorker` would be good. With `void $ forkIO`, we are just firing a thread in the wild and we have no control over it..